beacon-chain/rpc/service.go

```diff
@@ -320,7 +341,10 @@ func (s *Service) Stop() error {
 		log.Debug("Initiated graceful stop of gRPC server")
 	}
 	if s.slasherConn != nil {
-		s.slasherConn.Close()
+		if err := s.slasherConn.Close(); err != nil {
+			log.Error(err)
+			return err
```

Comment:
You should either return the error or log it out, I don't think you should ever be doing both.

Reply:
thanks